devops/helm/redis-cluster/values-0c27fb-dev.yaml

```diff
@@ -41,3 +41,16 @@ updateJob:
       memory: 1024Mi
 
   # resources: {}
+
+## Prometheus Exporter / Metrics
+##
+metrics:
+  ## @param metrics.enabled Start a side-car prometheus exporter
+  ##
+  enabled: true
+  ## Enable this if you're using https://github.com/coreos/prometheus-operator
+  ##
+  serviceMonitor:
```

Comment:
Is serviceMonitor required to have metrics enabled?

Reply:
yes to expose the service

Reply:
So, the answer to the question "Is serviceMonitor required to have metrics enabled?" is NO?

Reply:
Yes the answer is NO :)